Assignments/Program.cs

```diff
@@ -1 +1,35 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T14 - Reduced String
+// ---------------------------------------------------------------------------------------
+
+using System;
+using static System.ConsoleColor;
+namespace Spark;
+
+public class T14 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "String Reducer:-" + "\x1B[0m");
+      Console.WriteLine ("Here, the input is a string, the output is a reduced string of lowercase characters by deleting a pair of adjacent letters that match. ");
+      string[] testCases = { "aaabccddd", "aaaabbbb", "AAbcXXY", "xyz1233345667", "  " };
+      Console.WriteLine ("\n\x1B[4m" + "Test Cases:-" + "\x1B[0m");
+      for (int i = 0; i < testCases.Length; i++) {
+         Console.Write ($"\n{i + 1}. {testCases[i]} ==> ");
+         Console.WriteLine ($"{ReducedString (testCases[i])}", Console.ForegroundColor = Yellow);
+         Console.ResetColor ();
+      }
+      Console.Write ("\nNow, let's input enter a string to reduce: ");
+      string input = Console.ReadLine ();
+      Console.WriteLine ($"{ReducedString (input)}", Console.ForegroundColor = Yellow);

```

Comment:
You are using Console.WritLine (string format, object arg0) variant of WriteLine. It should be used like Console.WriteLine ("{0} is number, 1); You are assigning Console.ForegroundColor when calling the fuction so it is working. Remove it.